src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -36,14 +37,15 @@ export default class ReviewCollapsibleChapter extends React.Component {
 
   focusOnPage(key) {
     const name = `${key.replace(/:/g, '\\:')}`;
-    // legend & label target array type form elements
-    focusOnChange(name, 'p, legend, label');
+    focusElement(document.querySelector(`div[data-chapter="${name}"] button`));
   }
 
   handleEdit(key, editing, index = null) {
+    this.setState({ editing });
+    const chapterName = `${key}${index === null ? '' : index}`;
     this.props.onEdit(key, editing, index);
-    this.scrollToPage(`${key}${index === null ? '' : index}`);
-    this.focusOnPage(`${key}${index === null ? '' : index}`);
+    this.scrollToPage(chapterName);
+    this.focusOnPage(chapterName);
   }
```

Comment:
This will now update the `editing` state so we can set the appropriate button text. The `chapterName` const is just to consolidate the duplicate values into a single name.